llms/mlx_lm/generate.py

```diff
@@ -21,6 +21,17 @@ def setup_arg_parser():
         default="mlx_model",
         help="The path to the local model directory or Hugging Face repo.",
     )
+    parser.add_argument(
```

Comment:
I am not very sure if this is the best implementation for supporting enabling trust remote code and specifying a special token for the tokenizer.